lib/assets/javascripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "turbograft",
+  "version": "1.0.0",
+  "description": "Turbograft extends [Turbolinks](https://github.com/rails/turbolinks), allowing you to perform partial page refreshes.",
+  "scripts": {
+    "test": "dev test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/turbograft.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Shopify/turbograft/issues"
+  },
+  "homepage": "https://github.com/Shopify/turbograft#readme",
+  "dependencies": {
+    "promise-polyfill": "6.0.1"
```

Comment:
To expand, I could:
- either just hope the user's browser has `Promise` or the client of the library includes a polyfill
    - spoiler, phantomjs and other test environments don't, and selenium basically is nonfunctional for running them.
- polyfill promise myself?
- build an awkward bloated async foreach function and use that instead
- include a different async control library because it's 'not a polyfill' so it's ok (this has a much larger footprint if we use any of the established ones

I don't really think any of those make sense...